lib/core/notifiers/report_attendance_notifier.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:ata/core/notifiers/base_notifier.dart';
+import 'package:ata/core/services/user_service.dart';
+class AttendanceReportNotifier extends BaseNotifier {
+  final UserService _userService;
+
+  AttendanceReportNotifier(UserService userService) : _userService = userService;
+
+  Map attendanceRecordList = new Map();
+
+  //* all async request here
+  Future<void> refresh() async {
+    setBusy(true);
+    attendanceRecordList = await _userService.getDataAttendanceRecord();
+    await _userService.fetchDataToAttendanceRecord();
```

Comment:
already use attendanceRecordList , then await _userService.fetchDataToAttendanceRecord(); for ??